mypyc/genops.py

```diff
@@ -168,6 +170,9 @@ def __init__(self, types: Dict[Expression, Type], mapper: Mapper) -> None:
         # Maps unicode literals to the static c name for that literal
         self.unicode_literals = {}  # type: Dict[str, str]
 
+        # Maps integer literals to the static c name for that literal
+        self.integer_literals = {}  # type: Dict[int, str]
```

Comment:
Making a separate map is good for now, but once more sorts of literals get added it might be worth consolidating the map and logic into one path. (The map would have a type like `Dict[Union[int, str, float], str]`, maybe.)

Reply:
Acknowledged. I left a `TODO` comment in the code so that we remember to do this when we begin adding more dictionaries.